psycop/common/model_training_v2/problem_type/binary_classification.py

```diff
@@ -13,14 +13,22 @@
 from psycop.common.model_training_v2.problem_type.eval_dataset_base import (
     BinaryEvalDataset,
 )
+from psycop.common.model_training_v2.problem_type.problem_type_base import ProblemType
 from psycop.common.model_training_v2.training_method.base_training_method import (
     TrainingResult,
 )
 
-## Have .evaluate only return the main metric and df with pred_time_uuid,y_hat_probs
 
+def polarsframe_to_series(polarsframe: PolarsFrame) -> pl.Series:
```

Comment:
Neat solution to the issue of requiring different types of y for e.g. multilabel and binary classification. We should remember to write docstrings in e.g. BinaryClassification to specify that `y`should be a dataframe with a single column though

Reply:
Completely agree. Alternatively, they should not subclass from the same protocol. I've added an issue.